torchaudio/transforms.py

```diff
@@ -445,7 +445,7 @@ class MelSpectrogram(torch.nn.Module):
         n_mels (int, optional): Number of mel filterbanks. (Default: ``128``)
         window_fn (Callable[..., Tensor], optional): A function to create a window tensor
             that is applied/multiplied to each frame/window. (Default: ``torch.hann_window``)
-        power (float, optional): Exponent for the magnitude spectrogram,
+        power (float): Exponent for the magnitude spectrogram,
```

Comment:
we should leave in `optional` here because there is a default value. (I did not include `or None` when I updated MelSpectrogram documentation because of uncertainty so no change should be needed here)

Reply:
Good catch. Updated.